go.mod

```diff
@@ -20,28 +20,31 @@ require (
 	golang.org/x/crypto v0.45.0
 	golang.org/x/text v0.31.0
 	gopkg.in/ini.v1 v1.67.0
-	k8s.io/api v0.33.6
-	k8s.io/apiextensions-apiserver v0.33.6
-	k8s.io/apimachinery v0.33.6
-	k8s.io/client-go v0.33.6
-	k8s.io/code-generator v0.33.6
-	k8s.io/component-base v0.33.6
+	k8s.io/api v0.34.1
+	k8s.io/apiextensions-apiserver v0.34.1
+	k8s.io/apimachinery v0.34.1
+	k8s.io/client-go v0.34.1
+	k8s.io/code-generator v0.34.1
+	k8s.io/component-base v0.34.1
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/kube-openapi v0.0.0-20250610211856-8b98d1ed966a
+	k8s.io/kube-openapi v0.0.0-20250710124328-f3f2b991d03b
 	k8s.io/utils v0.0.0-20250604170112-4c0f3b243397
-	sigs.k8s.io/cluster-api v1.11.3
-	sigs.k8s.io/cluster-api/test v1.11.0
-	sigs.k8s.io/controller-runtime v0.21.0
-	sigs.k8s.io/structured-merge-diff/v4 v4.7.0
+	sigs.k8s.io/cluster-api v1.12.0-beta.0
```

Comment:
Not sure what the "process" is but should we maybe aim to go for the [beta-1 release](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.12.0-beta.1) directly?

( If it needs to be done but on a seperate PR, i can pick it up )

Reply:
Good point! I didn't check if there was a more recent release. I'll see if I can update this PR. It should be pretty easy.